js/ui/file_manager/ui.file_manager.editing.js

```diff
@@ -179,13 +174,18 @@ class FileManagerEditingControl extends Widget {
         this._showDialog(action.dialog, dialogArgumentGetter(items))
             .then(dialogResult => action.action(items, dialogResult))
             .then(result => {
-                whenSome(result,
+                whenSome(
+                    result,
                     () => this._raiseOnSuccess(action.getSuccessMessage(items), onlyFiles),
-                    info => {
-                        const fileItem = items[info.index];
-                        this._raiseOnError(info.errorId, fileItem);
-                    });
-            });
+                    info => this._onFileProviderError(info, items)
+                );
+            })
+            .fail(info => this._onFileProviderError(info, items));
```

Comment:
Can we use reject callback function instead of the separate `.fail()`? E.g.:
```
then(result => {
    whenSome(
        result,
        () => this._raiseOnSuccess(action.getSuccessMessage(items), onlyFiles),
        info => this._onFileProviderError(info, items)
	);
},
    info => this._onFileProviderError(info, items)
);
```

Reply:
Fixed